### PR TITLE
Correct the app-content-pages production assetPrefix to bypass CDN

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
app-content-pages

## Linked Issue and/or Talk Post
Follow-up to https://github.com/zooniverse/front-end-monorepo/pull/4238

## Describe your changes

Per [ADR 36](https://github.com/zooniverse/front-end-monorepo/blob/master/docs/arch/adr-36.md), use root URL  https://fe-content-pages.zooniverse.org/about/ for app-content-pages production site to bypass the CDN while debugging intermittent page loading issues.

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
- What user actions should my reviewer step through to review this PR?
- Which storybook stories should be reviewed?
- Are there plans for follow up PR’s to further fix this bug or develop this feature?

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)

### CODE REVIEW

File: packages/app-content-pages/next.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const assetPrefixes = {
   branch: 'https://fe-project-branch.preview.zooniverse.org/about',
   staging: 'https://frontend.preview.zooniverse.org/about',
```
Comment (goplayoutside3) [inline packages/app-content-pages/next.config.js:11]: Should the `staging` prefix be deleted similar to app-project, too?

Comment (mcbouslog) [reply packages/app-content-pages/next.config.js:11]: I don't think it's doing anything as is, but I suppose for consistency sake?

Comment (mcbouslog) [reply packages/app-content-pages/next.config.js:11]: I should have updated #4238 accordingly

**Review by mcbouslog — APPROVED** [review]

File: packages/app-content-pages/next.config.js (hunk 1 above)
Comment (goplayoutside3) [reply packages/app-content-pages/next.config.js:11]: Agreed, I'll push an update

**Pull request merged**